lib/pbench/server/api/resources/alembic/env.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
Is this from some boiler plate code?  If so, can we add comment or more doc string which references where it was taken from?

Reply:
Sure thing, I will update it. Btw just the background about this files: `alembic.ini` and `alembic` folder are an auto generated files when we run init alembic `alembic init alembic`. This files are need to be modified as per our needs and its encouraged by the alembic documentation, which I did with the `env.py` to get the postgres engine URI. 

The structure for the alembic migration files look like this:
```
yourproject/
    alembic/
        env.py
        README
        script.py.mako
        versions/
            <revision_ID>_<commit_message>.py
```